rfcs/0001-plugin-index.md

```diff
@@ -60,6 +60,7 @@ Each Plugin has its own Manifest, which contains a map with the following conten
     - `commit`: Clones `update_url` as git repository and checks `branch` for new commits.
   - `update_url`: A valid URL complementing `type`. May be omitted, in which case `homepage` shall be used instead.
   - `branch`: Allows for special branches to be used when `type` is `commit`. If omitted, the default branch will be used instead.
+  - `regex`: If `type` is `tag`, only tags whose name matches this regular expression will be considered. May be omitted.
```

Comment:
Based on which regex format?

Reply:
And just to confirm, we are doing a match instead of a search (so the entire text has to match the regex)? So in Hecter's example, it would be more like `regex: awesomeplugin.*`

Reply:
Sorry I missed that reply, probably because it and Hecter's comment got grouped into one notification.

That is a good question - I've started thinking of them kind of interchangeably, since different libraries use the terms differently... for instance, in python "match" means "search, but anchored at the start of the string".

Anyways, the way I see it, it's easier for everyone to require any part of the string to match. If you need to be explicit you can still use anchors: `regex: ^awesomeplugin.*$`

Reply:
```suggestion
  - `regex`: If `type` is `tag`, only tags whose name (fully or partially) matches this regular expression will be considered. Implementations should be PCRE2-compliant. May be omitted.
```